magprime/templates/regextra.html

```diff
@@ -13,6 +13,9 @@
             $("input:radio[name='amount_extra'][value=250]").parents('.btn').append($('#canvas_print'));
             $('#canvas_print').on('click', function(e) { e.stopPropagation(); })
         }
```

Comment:
Boooooo. Booooooooooooooooooooooo.

Thanks for pointing this out - I'll fix it tomorrow.